docs/ovn_k8s.conf.5

```diff
@@ -23,45 +22,66 @@ This represents the conntrack zone used for the conntrack flow rules.
 .PP
 .SH [Logging]
 .TP
-\fBlogfile\fR=/var/log/openvswitch/ovn-k8s-cni-overlay.log
-Logfile name (with path) for ovnkube to write to.
+\fBlogfile\fR=
+When logging to a file, this is the path and file. E.g., /var/log/openvswitch/ovn-k8s-cni-overlay.log
+Default is not to log to a file.
 .TP
-\fBloglevel\fR=0
-Level is the logging verbosity level: 5=debug, 4=info, 3=warn, 2=error, 1=fatal (default: 0).
+\fBloglevel\fR=4
+Level is the logging verbosity level: 5=debug, 4=info, 3=warn, 2=error, 1=fatal (default: 4).
 .SH [CNI]
 .PP
 The following config values are used for the CNI plugin.
 .TP
-\fBcni-conf-dir\fR=/etc/cni/net.d
+\fBconf-dir\fR=/etc/cni/net.d
 Directory that contains the cni plugin.
 .TP
-\fBcni-plugin\fR=ovn-k8s-cni-overlay
+\fBplugin\fR=ovn-k8s-cni-overlay
 Cni plugin name.
 .SH [Kubernetes]
 .PP
-K8S apiserver and authentication details are stored in the following options.
+K8S apiserver and authentication details are declared in the following options.
 .TP
 \fBkubeconfig\fR=
 Absolute path to the kubeconfig file (not required if the apiserver, cacert, and token are given).
 .TP
-\fBcacert\fR=/etc/openvswitch/k8s-ca.crt
-The absolute path to the Kubernetes API CA certificate (not required if kubeconfig is given).
+\fBcacert\fR=
+The absolute path to the Kubernetes API CA certificate, e.g.,
+/etc/openvswitch/k8s-ca.crt (not required if kubeconfig is given).
 .TP
-\fBapiserver\fR=
-URL to the Kubernetes apiserver (default "https://localhost:8443") (not required if kubeconfig is given).
+\fBapiserver\fR=http://localhost:8080
+URL to access the Kubernetes apiserver. The host must be a name, not IP. The kubernetes/openshift
+api serveri, by default, listens on 8443. E.g., https://masterhost:8443"
+(Not required if kubeconfig is given.)
 .TP
 \fBtoken\fR=
-The Kubernetes API authentication token (not required if kubeconfig is given).
+The Kubernetes API authentication token permits the ovn plugin to communicate with the
```

Comment:
We need the kube way here definitely. We can provide the extra things that we need to do on openshift (e.g. scc anyuid is openshift specific I believe)

Reply:
@rajatchopra Do you know the kubectl commands. I have not used kubernetes, just Openshift.